src/lib/stream/ctr/ctr.cpp

```diff
@@ -125,4 +125,9 @@ void CTR_BE::increment_counter()
    m_pad_pos = 0;
    }
 
+void CTR_BE::seek(u64bit offset)
+   {
+      (void)offset;
```

Comment:
CTR should throw Not_Implemented("CTR_BE::seek") - CTR supports this but it does not have to be added in this commit.